README.md

```diff
@@ -1,29 +1,38 @@
 <img src="https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg" height="64px"/>
 
 - [OpenSearch Project Helm-Charts](#helm-charts)
-- [Installation](#installation)
 - [Status](#status)
+- [Installation](#installation)
+- [Change Logs](#change-logs)
 - [Contributing](#contributing)
 - [Getting Help](#getting-help)
 - [Code of Conduct](#code-of-conduct)
 - [Security](#security)
 - [License](#license)
-- [Copyright](#copyright)
 
-## OpenSearch Project Helm-Charts
+## Helm-Charts
 
 A community repository for Helm Charts of OpenSearch Project.
 
+## Status
+
+![Testing](https://github.com/opensearch-project/helm-charts/workflows/Lint%20and%20Test%20Charts/badge.svg)
+![Release](https://github.com/opensearch-project/helm-charts/workflows/Release%20Charts/badge.svg)
+
 ## Installation
 
-[OpenSearch Helm chart](https://opensearch.org/docs/opensearch/install/helm/)
+```shell
+helm repo add opensearch https://opensearch-project.github.io/helm-charts/
+helm repo update
+```
 
-[OpenSearch Dashboards Helm chart](https://opensearch.org/docs/dashboards/install/helm/)
+You can then run `helm search repo opensearch` to see the charts.
```

Comment:
IMO , lets change the README in a separate PR after verifying the `helm search` command. I know it will work but still to prevent any dangling pointers it will be good.

Reply:
@TheAlgo I reverted the instructions to their previous value.